src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer.java

```diff
@@ -47,12 +47,14 @@ public KubernetesComputer(KubernetesSlave slave) {
     @Override
     public void taskAccepted(Executor executor, Queue.Task task) {
         super.taskAccepted(executor, task);
-        LOGGER.fine(" Computer " + this + " taskAccepted");
+        Queue.Executable exec = executor.getCurrentExecutable();
+        LOGGER.log(Level.FINE, " Computer " + this + " accepted task " + exec);
```

Comment:
BTW sub-`INFO` logger calls should either use the `Supplier` overload or message formats

```java
LOGGER.log(Level.FINE, " Computer {0} accepted task {1}", new Object[] {this, exec});
```

to avoid string concatenation when unused. Unlikely to matter much in this case.

Reply:
[PR 564](https://github.com/jenkinsci/kubernetes-plugin/pull/564)

Reply:
https://github.com/jenkinsci/kubernetes-plugin/pull/564 that is. FYI: GitHub has nice handling (hover tips, backlinks) for several kinds of URLs when pasted directly into GH-flavored Markdown, but _not_ when used in a Markdown link.